pages/dkp/kommander/2.2/release-notes/index.md

```diff
@@ -24,17 +24,14 @@ This version supports Kubernetes versions between 1.21.0 and 1.22.x. Any cluster
 
 ### New features and capabilities
 
-#### Feature 1, 2, etc.
+#### Upgrade catalog application via CLI and UI
 
+You can use either the CLI or the UI to [upgrade your catalog applications](..\projects\applications\catalog-applications\index.md).
```

Comment:
```suggestion
You can use either the CLI or the UI to [upgrade your catalog applications](/projects/applications/catalog-applications/#upgrade-catalog-applications).
```

Reply:
Thanks Casey, 101 ways to do the links always throws me off. I thought the backwards slashed looked weird - FWIW, that's how they copied down directly from Git, as a relative link!